hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/pipeline/ListPipelinesSubcommand.java

```diff
@@ -38,11 +38,34 @@
   @CommandLine.ParentCommand
   private SCMCLI parent;
 
+  @CommandLine.Option( names = {"-ff", "--filterByFactor"},
+          description = "Filter listed pipelines by Factor(ONE/one)", defaultValue = "",
+          required = false)
+  private String factor;
+
+  @CommandLine.Option( names = {"-fs", "--filterByState"},
```

Comment:
Nit. Maybe some other shorthand than "-fs", since it usually means filesystem in HDFS world.

Reply:
Ok. I update to -ffc and -fst.